Thanks for your pull request!

But unfortunately your code causes a compiler warning using GCC 4.8 when compiling C code:

```
gcc   -m32 -g -O3 -Wall -pedantic -Wextra -Werror -Wno-unused-parameter -Wno-long-long  -O3 -DNDEBUG test/test2.c
/home/travis/build/kimwalisch/libpopcnt/./libpopcnt.h:816:3: error: C++ style comments are not allowed in ISO C90 [-Werror]
 } // extern "C"
```

So can you please remove the ```// extern "C"``` comment?

--- a/libpopcnt.h
+++ b/libpopcnt.h
@@ -813,7 +813,7 @@ static inline uint64_t popcnt(const void* data, uint64_t size)
 #endif
 
 #ifdef __cplusplus
-} // extern "C"
+} /* extern "C" */
 #endif
 
 #endif /* LIBPOPCNT_H */

